### PR TITLE
Feature/configure open source file

### PR DESCRIPTION
**Pull request by juan-campuzano**
se agregan pipelines y archivos open source

### CODE REVIEW

File: lib/src/helpers/local_file_comparator_with_threshold.dart [hunk 1]
```diff
@@ -6,6 +6,8 @@ import '../golden_configuration.dart';
 /// This class is intended to use a custom value of difference acceptance
 /// so it will take the path to the test and if the value is more than
 /// [_kGoldenTestsThreshold] it will fail the test.
+
+// coverage:ignore-start
```
Comment (santitigaga) [inline lib/src/helpers/local_file_comparator_with_threshold.dart:10]: why ignore that?

File: example/test/src/presentation/home/goldens/golden_iphone_14_pro.png
Comment (santitigaga) [inline example/test/src/presentation/home/goldens/golden_iphone_14_pro.png:1]: please change the image

File: example/test/src/presentation/home/goldens/golden.png
Comment (santitigaga) [inline example/test/src/presentation/home/goldens/golden.png:1]: please change the image

**Review by santitigaga — CHANGES_REQUESTED** [review]

File: example/test/src/presentation/home/goldens/golden_iphone_14_pro.png
Comment (juan-campuzano) [reply example/test/src/presentation/home/goldens/golden_iphone_14_pro.png:1]: fixed

**Review by santitigaga — APPROVED** [review]

**Pull request merged**